src/iframeMessenger.js

```diff
@@ -423,6 +423,47 @@
             }
         }
 
+        function _addAcquisitionDataToLinks(acquisitionData) {
+            var links = document.getElementsByClassName('js-acquisition-link');
+            var json = encodeURIComponent(JSON.stringify(acquisitionData));
+            for (var i = 0; i < links.length; i++) {
+                var href = link.attr('href');
+                if (href) {
+                    href = `${href}${href.includes('?') ? '&' : '?'}${json}`;
```

Comment:
sorry... `.includes()` isn't available in IE, you'll have do `indexOf('?') > -1`